plugins/circleci/src/pages/DetailedViewPage/DetailedViewPage.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { FC } from 'react';
+import { Content, InfoCard, useApi } from '@backstage/core';
+import { Grid, Box } from '@material-ui/core';
+import { PluginHeader } from 'components/PluginHeader';
+import { BuildWithSteps, BuildStepAction } from 'circleci-api';
+import { circleCIApiRef } from 'api';
+import { useParams } from 'react-router-dom';
+import { ActionOutput } from './lib/ActionOutput/ActionOutput';
+import { Layout } from 'components/Layout';
+import { Dispatch, iRootState } from 'state/store';
+import { useDispatch, useSelector } from 'react-redux';
+
+const BuildName: FC<{ build: BuildWithSteps | null }> = ({ build }) => (
+  <>
+    #{build?.build_num} - {build?.subject}
+  </>
+);
+import { makeStyles } from '@material-ui/core/styles';
+const useStyles = makeStyles((theme) => ({
+  neutral: {},
+  failed: {
+    position: 'relative',
+    '&:after': {
+      pointerEvents: 'none',
+      content: '""',
+      position: 'absolute',
+      top: 0,
+      right: 0,
+      left: 0,
+      bottom: 0,
+      boxShadow: `inset 4px 0px 0px ${theme.palette.error.main}`,
+    },
+  },
+  running: {
+    position: 'relative',
+    '&:after': {
+      pointerEvents: 'none',
+      content: '""',
+      position: 'absolute',
+      top: 0,
+      right: 0,
+      left: 0,
+      bottom: 0,
+      boxShadow: `inset 4px 0px 0px ${theme.palette.info.main}`,
+    },
+  },
+  cardContent: {
+    backgroundColor: theme.palette.background.default,
+  },
+  success: {
+    position: 'relative',
+    '&:after': {
+      pointerEvents: 'none',
+      content: '""',
+      position: 'absolute',
+      top: 0,
+      right: 0,
+      left: 0,
+      bottom: 0,
+      boxShadow: `inset 4px 0px 0px ${theme.palette.success.main}`,
+    },
+  },
+}));
+
+const pickClassName = (
+  classes: ReturnType<typeof useStyles>,
+  build: BuildWithSteps = {} as BuildWithSteps,
+) => {
+  switch (true) {
```

Comment:
I think just if/else would be clearer here

Reply:
- [x] if/else